tyk-hybrid/templates/deployment-gw-repset.yaml

```
@@ -78,17 +78,19 @@ spec:
           - name: TYK_GW_HTTPSERVEROPTIONS_USESSL
             value: "{{ .Values.gateway.tls }}"
           - name: TYK_GW_SLAVEOPTIONS_RPCKEY
-            value: "{{.Values.tyk_k8s.org_id }}"
+            value: "{{.Values.rpc.rpcKey }}"
```

Since the variable has changed, something like this should keep compat:
```suggestion
            value: "{{ default .Values.tyk_k8s.org_id .Values.rpc.rpcKey }}"
```